bebop101:
  initial_pose:
    x: -1.7
    y: 0
    z: 1
bebop102:
  initial_pose:
    x: 1.7
    y: 0
    z: 1
field:
  mesh_acc: [200,200] #x,tan(theta)
  limit: [[-2,2],[-2,2]] #[x[min,max], tan(theta)[min,max]]
